components/style/values/specified/border.rs

```diff
@@ -171,3 +172,47 @@ impl Parse for BorderSpacing {
         }).map(GenericBorderSpacing)
     }
 }
+
+define_css_keyword_enum! { RepeatKeyword:
+    "stretch" => Stretch,
+    "repeat" => Repeat,
+    "round" => Round,
+    "space" => Space
+}
+
+/// The specified value for the `border-image-repeat` property.
+///
+/// https://drafts.csswg.org/css-backgrounds/#the-border-image-repeat
+#[derive(Clone, Debug, MallocSizeOf, PartialEq, ToCss)]
+pub enum BorderImageRepeat {
+    /// `[ stretch | repeat | round | space ]{1,2}`
+    Keywords(RepeatKeyword, Option<RepeatKeyword>),
```

Comment:
This is just an enum with a single variant, so this can be a `struct BorderImageRepeat(RepeatKeyword, Option<RepeatKeyword>)`, right?